File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

/**
 * 品牌接口
 * @author Administrator
 *
 */
public interface BrandService {

	List<TbBrand> findAll();

	PageResult findPage(Integer page, Integer size);

	void add(TbBrand tbBrand);

	TbBrand findOne(Long id);

	void update(TbBrand tbBrand);

	void delete(Long[] ids);

	PageResult findByNameAdFirstChar(TbBrand tbBrand,Integer page, Integer size);


}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    };

    this.findPage=function (page,size) {
        return $http.get('../brand/findPage.do?page='+page+'&size='+size);
    };

    this.add=function (entity) {
        return $http.post('../brand/add.do',entity);
    };

    this.update=function (entity) {
        return $http.post('../brand/update.do',entity);
    };

    this.delet=function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds);
    };

    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    };

    this.search=function (page,size,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
    }

    //查询下拉列表数据
    this.findBrandList=function () {
        return $http.post('../brand/findOptionList.do');
    }
});